lxd-agent/main_agent.go

```diff
@@ -166,6 +160,21 @@ func (c *cmdAgent) Run(cmd *cobra.Command, args []string) error {
 		}
 	}()
 
+	// Check whether we should start the devlxd server in the early setup. This way, /dev/lxd/sock
+	// will be available for any systemd services starting after the lxd-agent.
+	if shared.PathExists("devlxd") {
```

Comment:
Lets call this agent.conf and then put a devlxd specific json key inside it, so we can expand this for other early start features in the future.